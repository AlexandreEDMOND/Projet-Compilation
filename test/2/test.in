int main()
{
  int a = 8;
  int b;
  b = a + 1;
  int c = 5*b;
  c++;
  a = b - a;
  a = c / 3;
  int d = -a;
  return 0;
}