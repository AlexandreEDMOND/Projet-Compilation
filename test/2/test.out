Nouvelle valeur par 8.000000 + 1.000000 : 9.000000
Nouvelle valeur par 5.000000 * 9.000000 : 45.000000
Nouvelle valeur par 9.000000 - 8.000000 : 1.000000
Nouvelle valeur par 46.000000 / 3.000000 : 15.333333
Programme correctement compil√©
